cime_config/config_grids.xml

```diff
@@ -2569,8 +2579,8 @@
     <domain name="ne1024np4.pg2">
       <nx>25165824</nx>
       <ny>1</ny>
-      <file grid="atm|lnd" mask="oRRS18to6v3">$DIN_LOC_ROOT/share/domains/domain.lnd.ne1024pg2_oRRS18to6v3.200212.nc</file>
-      <file grid="ice|ocn" mask="oRRS18to6v3">$DIN_LOC_ROOT/share/domains/domain.ocn.ne1024pg2_oRRS18to6v3.200212.nc</file>
+      <file grid="atm|lnd" mask="oRRS18to6v3">$DIN_LOC_ROOT/share/domains/domain.lnd.ne1024pg2_oRRS18to6v3.210523.nc</file>
```

Comment:
Why do these need to be updated? Did the domain files change for some reason? I think these should be the same.

Reply:
Good point, Ben.  I was curious . There were a  number of domain files that have the same time stamp (200212). But the files for ne1024pg2_oRRS18to6v3 do not exist in the inputdata (not on cori where I did the test).  Just check on the inputdata server. The 200212 domain files are available there. I will revert this change.